packages/babel-plugin-proposal-decorators/test/fixtures/2023-11-classes/.replacement-static-private-environment-super-2/exec.js

```diff
@@ -1,34 +0,0 @@
-{
```

Comment:
This test is moved to the sub test "different private/super access in a non-decorated static method" in `replacement-static-private-environment-super/exec.js`.